### DISCUSSION TITLE
How do I make the text bubbles full width?

### DISCUSSION THREAD

**Discussion by kevintran-git** [flyerhq/flutter_chat_ui#451] [Ideas & Requests]
I'm looking to use FlyerChat as a frontend for a Large Language Model. I would like to design the bubbles to:
1. Not alternate sides
2. Take up the full width of the screen

Some examples of these are:
<img width="264" alt="image" src="https://github.com/flyerhq/flutter_chat_ui/assets/26168327/dfa42d2e-f07d-4502-bf95-caa25a81167c">
<img width="245" alt="image" src="https://github.com/flyerhq/flutter_chat_ui/assets/26168327/184c7b77-8e1d-415f-92ec-9c4d09e6e07f">
<img width="180" alt="image" src="https://github.com/flyerhq/flutter_chat_ui/assets/26168327/d9114a1c-949d-4c01-bac5-0c0cf88ea0bb">

Would I be able to accomplish this with the Flyer Chat UI? Otherwise, I love the polish and the animations that this library has!

**Comment by demchenkoalex** [comment]
Hello! Not in the current version, but I will add this to the list to include in v2. I will be creating a 1 big issue with everything people need to track progress of v2, and will link It here when created.

**Reply by avocadohooman** [reply]
Has this been implemented in v2?

**Reply by demchenkoalex** [reply]
This is a default behaviour in v2, yes
